corehq/apps/smsbillables/migrations/0005_add_zero_http_gateway_fee_prep.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.apps.sms.models import OUTGOING
+from corehq.apps.smsbillables.management.commands.add_zero_http_gateway_fee import \
+    add_zero_http_gateway_fee
+
+
+def add_zero_http_gateway_fee_prep(apps, schema_editor):
+    add_zero_http_gateway_fee('635b73026a9a8eab55c77a84bccf6be2', OUTGOING, apps)
```

Comment:
Yeah, I think you're right. There's one small tweak that I think will be necessary before we charge this one - let's close this PR, I'll make the tweak, and we can reopen this with the right rates for this specific backend.